lua/nvim-lsp-ts-utils/import-all.lua

```diff
@@ -177,30 +177,53 @@ local response_handler_factory = function(callback)
     end
 end
 
+local should_reorder = function(edits)
+    local modules = {}
+    for _, edit in ipairs(edits) do
+        if edit.newText then
+            local module = string.match(edit.newText, "import.*from (.+)[;\n]?")
+
+            if not module then
+                return
+            end
+
+            local import = module:gsub("\n", "")
```

Comment:
```suggestion
            local source = vim.trim(module)
```

I think I suggested this before, but I think changing the variable name to `source` and using `vim.trim()` makes it a little easier to parse this at a glance. (Of course, the other uses of the variable have to be updated, too.)